ksim/task/rl.py

```
@@ -590,6 +590,7 @@ def step_engine(
         trajectory = Trajectory(
             qpos=next_physics_state.data.qpos,
             qvel=next_physics_state.data.qvel,
+            actuator_frc=next_physics_state.data.actuator_force,
```

oh... maybe we should figure out a better way to populate the trajectory during runtime... will noodle on this

edit: maybe the best way to do this is to check `obs` for a particular key - the reward here should throw an error if the relevant key does not exist. so, we should add some `obs` which gets `actuator_force`

---

What about information that is not observation available at run time? I guess we can always add it to the critic?